spec/features/activities/work_package/activities_spec.rb

```diff
@@ -980,18 +980,224 @@
       # we need to wait a bit before triggering the update below
       # otherwise the update is already picked up by the initial (async) workpackage attributes update called in the connect hook
       # and we wouldn't test the polling based update below
-      sleep 2
+      sleep 2 # rubocop:disable OpenProject/NoSleepInFeatureSpecs
```

Comment:
🔴  we since added [rspec-wait](https://github.com/laserlemon/rspec-wait) in https://github.com/opf/openproject/pull/16761 and can already leverage `wait_for` in place of static sleep times.

Ex. https://github.com/opf/openproject/blob/dev/spec/features/activities/work_package/activities_spec.rb#L373